pkgs/games/dwarf-fortress/default.nix

```diff
@@ -18,6 +21,37 @@ let
     };
   };
 
+  dfhack = import ./dfhack.nix {
+    inherit stdenv cmake zlib perl XMLLibXML XMLLibXSLT;
+    version = "0.40.24-r3";
+    src = fetchgit {
+      url = "https://github.com/DFHack/dfhack.git";
+      rev = "0849099f2083e100cae6f64940b4eff4c28ce2eb";
+      sha256 = "0lnqrayi8hwfivkrxb7fw8lb6v95i04pskny1px7084n7nzvyv8b";
+    };
+  };
+  dfhack_installPhase = ''
+    cat > $out/bin/dfhack_install_dir <<EOF
+    #!/bin/sh
+    test -z "\$1" && echo "This creates a Dwarf Fortress/DFHack game directory. Please specify a directory (preferably empty or non-existent) for this." && exit 1
```

Comment:
Why do we need to create a different directory outside the nix-store for dfhack?